pkg/types/tuf/v0.0.1/tuf_key_v0_0_1_schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.dev/types/tuf/tuf.json",
+    "title": "TUF signature v0.0.1 Schema",
+    "description": "Schema for tuf signature verification data (keys and threshold)",
+    "type": "object",
+    "properties": {
+        "keys": {
+            "description": "public key information",
+            "type": "object",
+            "properties": {
+                "keyid": {
```

Comment:
should this be `keyid_hash_algorithms` and be changed to be an array of enum values?

Reply:
resolved by proxy of deletion :D